W1-M3-branches-and-loops-local/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+//ExploreIf();
+
+void ExploreIf()
+{
+  int a = 5;
+  int b = 3;
+  if (a + b > 10)
+  {
+    Console.WriteLine("The answer is greater than 10");
+  }
+  else
+  {
+    Console.WriteLine("The answer is not greater than 10");
+  }
+
+  int c = 4;
+  if ((a + b + c > 10) && (a > b))
+  {
+    Console.WriteLine("The answer is greater than 10");
+    Console.WriteLine("And the first number is greater than the second");
+  }
+  else
+  {
+    Console.WriteLine("The answer is not greater than 10");
+    Console.WriteLine("Or the first number is not greater than the second");
+  }
+
+  if ((a + b + c > 10) || (a > b))
+  {
+    Console.WriteLine("The answer is greater than 10");
+    Console.WriteLine("Or the first number is greater than the second");
+  }
+  else
+  {
+    Console.WriteLine("The answer is not greater than 10");
+    Console.WriteLine("And the first number is not greater than the second");
+  }
+}
+
+/*

```

Comment:
Another convention, as we use versioning is, if you don't need the code either remove it or provide details why it is commented out. This improves team readability!